api-v3-stable.rst

```diff
@@ -0,0 +1,48 @@
+.. post:: Feb 1, 2021
+   :tags: api, feature, reference
+   :author: Manuel
+   :location: BCN
+
+API v3 is now stable!
+=====================
+
+We are excited to announce that our API v3 has reached its stable version,
+and it's now available for all Read the Docs users.
+This includes Read the Docs Community and Read the Docs for Business.
+Since we :doc:`announced the beta version </api-v3-beta>` of it,
+we have been adding extra functionality and bug-fixing some minor issues based on users feedback.
+
+The new API v3 *is not* a fully replacement (yet!) of API v2,
+but **we highly recommend using API v3 for all the new code you write from now on**.
+API v2 is planned to be deprecated soon,
+though we have not yet set a time frame for deprecation.
+We will alert users with our plans when we do.
+
+
+Nice features
+-------------
+
+These are some of the possible actions API v3 allows and we like the most,
+
+- Ability to import new projects
+- Activate a version
+- Trigger builds
+- Check build status
+- Manage redirects
+
+They allow you to easily manage common tasks like setup the documentation for a new project,
+automate the release of a new version, or even pause it if you detect that the build failed for some reason.
+Hook the rename of a file in your VCS to the creation of a redirect in its docs,
+and more!
```

Comment:
I feel like this section needs to be written out a bit more. It feels weird to have the bullet points and then also a paragraph with similar content. We should likely remove the bulleted list and talk about a couple common use cases.

Reply:
My idea here was to highlight some interesting features (to make the reader match them into their own use cases) and then combine those features into useful real-life use cases we already known. Makes sense? Any suggestion about how I can achieve that without sounding repetitive here?

Reply:
It is basically just another list, so we should either make it a list, or fully explain the use cases. I'll push up an update making it a list, since that seems better for now.